app/controllers/chatbot_controller.rb

```diff
@@ -0,0 +1,20 @@
+class ChatbotController < ApplicationController
+  def show
+    # TODO order by priority and maybe have a fallback
+    chatbot_channel = Channel.find_by(name: "chatbot")
+    current_report = Report.current
+    tc = TextComponent
+      .where(report: current_report)
+      .where(topic: Topic.find_by(name: params[:topic]))
+      .joins(:channels)
+      .where("channels.id = :channel_id", {channel_id: chatbot_channel.id})
```

Comment:
This seems complicated to me. I would prefer `current_report.active_text_components(topic: topic)`